gettsim/taxes/calc_taxes.py

```diff
@@ -70,7 +78,7 @@ def abgeltung(tax_unit, tb):
                 tax_unit["gross_e5_tu"] - 2 * (tb["spsparf"] + tb["spwerbz"]), 0
             )
         )
-    return tax_unit_abgelt["abgst"]
+    return tax_unit_abgelt["abgst"].round(2)
```

Comment:
As HMG pointed out, we shouldn't round in our code. Sorry didn't saw that in the first round!